hack/genref/config.yaml

```diff
@@ -0,0 +1,19 @@
+hiddenMemberFields:
```

Comment:
Can you add a comment with a link to the documentation for this config file, so we can easily figure out how to update it later if we need to?